dashboard/src/components/dashboard/GuildCard.vue

```diff
@@ -25,7 +25,7 @@
 				<strong class="is-size-5">{{ guild.name }}</strong>
 			</div>
 		</div>
-		<footer class="card-footer is-size-6">
```

Comment:
You can actually have multiple footers on a page.
See https://www.w3schools.com/tags/tag_footer.asp and https://bulma.io/documentation/components/card/

Reply:
Point taken. Reverting it now.